its/tests/src/test/java/its/ConnectedModeTest.java

```diff
@@ -114,6 +115,7 @@
     .addPlugin(MavenLocation.of("org.sonarsource.slang", "sonar-kotlin-plugin", System.getProperty("kotlinVersion")))
     .addPlugin(MavenLocation.of("org.sonarsource.slang", "sonar-ruby-plugin", System.getProperty("rubyVersion")))
     .addPlugin(MavenLocation.of("org.sonarsource.slang", "sonar-scala-plugin", System.getProperty("scalaVersion")))
+    .addPlugin(MavenLocation.of("org.sonarsource.slang", "sonar-go-plugin", System.getProperty("goVersion")))
```

Comment:
For the system property to exist, you have to update also https://github.com/SonarSource/sonarlint-core/blob/master/run-integration-tests.sh